configs/trainer/default.yaml

```diff
@@ -10,3 +10,4 @@ weights_summary: null
 progress_bar_refresh_rate: 5
 resume_from_checkpoint: null
 fast_dev_run: false
+profiler: 'simple'
```

Comment:
@jacobbieker this should do it, i.e similar to 
trainer = Trainer(..., profiler="simple")

Reply:
Perfect!